action.yaml

```diff
@@ -0,0 +1,65 @@
+name: 'GTA'
+description: 'Finds Go packages that have deviated from their upstream source.'
+
+inputs:
+  base:
+    description: 'Base branch to diff against'
+    required: false
+    default: 'origin/master'
+  include:
+    description: 'Define changes to be filtered with a set of comma-separated prefixes'
+    required: false
+    default: ''
+  merge:
+    description: 'Diff using the latest merge commit'
+    required: false
+    default: false
+  json:
+    description: 'Output list of changes as JSON'
+    required: false
+    default: false
+  buildable_only:
+    description: 'Keep buildable changed packages only'
+    required: false
+    default: true
+  changed_files:
+    description: 'Path to a file containing a newline-separated list of files that have changed'
+    required: false
+    default: ''
+  tags:
+    description: 'A list of build tags to consider'
+    required: false
+    default: ''
+
+runs:
+  using: "composite"
+  steps:
+    - name: Set up Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: '1.24'
+
+    - name: Install Go package
+      shell: bash
+      # todo nickyd: update to current stable version tag
+      run: go install github.com/digitalocean/gta/cmd/gta@latest
+
+    - name: Run GTA
+      shell: bash
+      run: |
+        go run ./cmd/gta/main.go \
```

Comment:
if you're doing `go install`, there's no need to do `go run` as you can run the compiled `$GOBIN/gta` directly

alternatively, `go run ./cmd/gta` would also make sense if you want to run a specific commit as opposed to release (`@latest`) 
either way, we don't need both imo

Reply:
Good callout, I need to update that.

I don't think `go run ./cmd/gta` would work as a good alternative as I understand it, since it would require cloning this repo into the caller workflow, instead of just targeting the action.

Anyway, I'm just hacking away rn and opened this draft PR to test the workflow. I'll let you know when this is in an okay state 😄